### ISSUE TITLE
Can the rest API of swagger UI be exported? 

### ISSUE THREAD

**Issue by qwyinxiang** [thingsboard/thingsboard#1146]
Can the rest API be exported in yaml or json format?
![image](https://user-images.githubusercontent.com/31084798/46859375-2a6fd100-ce40-11e8-812e-47a1444a90b4.png)

**Comment by ChristianLutz** [comment]
Yes, just use <YOUR_URL>/v2/api-docs?group=thingsboard now just copy and paste into swagger.json

**Comment by hieunvce** [comment]
Note for other people,

You can get the URL here: https://<YOUR_URL>/swagger-ui/

![scrshot](https://github.com/thingsboard/thingsboard/assets/29145542/f37d1b99-2722-4012-851c-69ae0adaecb2)

Currently, it is: https://cloud.thingsboard.io/v3/api-docs?group=thingsboard
